sonar-text-dotnet/pipeline_scripts/analysis_begin_step.ps1

```diff
@@ -1,12 +1,12 @@
 Set-Location $env:PROJECT_DIR
 
-if ($env:CIRRUS_PR -eq "") {
+if ($env:CIRRUS_BRANCH -eq "master") {
```

Comment:
this will again handle only the master. what about the other feature branches?

Reply:
Fixed.